migrations/20231210123554_init.sql

```diff
@@ -0,0 +1,236 @@
+(
+    id          UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    -------------------------------------------------------
+    name        VARCHAR(255) NOT NULL,
+    description TEXT,
+    website     VARCHAR(255),
+    with_staff  BOOLEAN NOT NULL,
+    start_date  DATE NOT NULL,
+    end_date    DATE NOT NULL,
+    avatar_url  VARCHAR(255),
+    -------------------------------------------------------
+    created_at  TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at   TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at  TIMESTAMP
+);
+
+
+CREATE TABLE IF NOT EXISTS "Timesheet"
+(
+    id          UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    -------------------------------------------------------
+    start_date  DATE NOT NULL,
+    end_date    DATE NOT NULL,
+    total_hours FLOAT NOT NULL,
+    is_editable BOOLEAN NOT NULL,
+    note        TEXT,
+    -------------------------------------------------------
+    created_at  TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at   TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at  TIMESTAMP,
+    -------------------------------------------------------
+    user_id     UUID,
+    company_id  UUID,
+    event_id    UUID,
+    -------------------------------------------------------
+    FOREIGN KEY (user_id) REFERENCES "User" (id),
+    FOREIGN KEY (company_id) REFERENCES "Company" (id),
+    FOREIGN KEY (event_id) REFERENCES "Event" (id)
+);
+
+
+CREATE TABLE IF NOT EXISTS "WorkDay"
+(   
+    date        DATE NOT NULL,
+    total_hours FLOAT NOT NULL,
+    comment     TEXT,
+    is_editable BOOLEAN NOT NULL,
+    -------------------------------------------------------
+    created_at  TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at   TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at  TIMESTAMP,
+    -------------------------------------------------------
+    timesheet_id UUID,
+    -------------------------------------------------------
+    FOREIGN KEY (timesheet_id) REFERENCES "Timesheet" (id)
+);
+
+
+CREATE TABLE IF NOT EXISTS "EventStaff"
+(
+    id          UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    -------------------------------------------------------
+    level      "EmployeeLevel" NOT NULL,
+    -------------------------------------------------------
+    created_at  TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at   TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at  TIMESTAMP,
+    -------------------------------------------------------
+    user_id     UUID,
+    event_id    UUID,
+    -------------------------------------------------------
+    FOREIGN KEY (user_id) REFERENCES "User" (id),
+    FOREIGN KEY (event_id) REFERENCES "Event" (id)
+);
+
+
+CREATE TABLE IF NOT EXISTS "Task"
+(
+    id          UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    -------------------------------------------------------
+    title       VARCHAR(255) NOT NULL,
+    description TEXT,
+    finished_at DATE,
+    priority    "TaskPriority" NOT NULL,
+    accepted    "AcceptanceStatus" NOT NULL,
```

Comment:
Again, this should be something like 'accepts_staff' instead. It says about whether the task is accepting new staff. 
Also should be a boolean value. 'AcceptanceStatus' relates to assigned_staff and event_staff.